src/transformers/models/albert/configuration_albert.py

```diff
@@ -103,51 +115,47 @@ class AlbertConfig(PretrainedConfig):
     >>> configuration = model.config
     ```"""
 
+    vocab_size: int = validated_field(strictly_positive_int, default=30000)
```

Comment:
Hmm, not sure how we could do better but I start to realize this is a very verbose way of defining things... :confused: I'll think about it but happy to receive suggestions if any